client/asset/interface.go

```diff
@@ -515,9 +515,12 @@ type Wallet interface {
 	// payment. This method need not be supported by all assets. Those assets
 	// which do no support DEX registration fees will return an ErrUnsupported.
 	RegFeeConfirmations(ctx context.Context, coinID dex.Bytes) (confs uint32, err error)
+	// TransactionConfirmations gets the number of confirmations for the specified
+	// transaction.
+	TransactionConfirmations(ctx context.Context, txID string) (confs uint32, err error)
 	// Send sends the exact value to the specified address. This is different
 	// from Withdraw, which subtracts the tx fees from the amount sent.
-	Send(address string, value, feeRate uint64) (Coin, error)
+	Send(address string, value, feeRate uint64) (txID string, coin Coin, err error)
```

Comment:
Instead of adding new return args, please try this https://github.com/buck54321/dcrdex/compare/a164e13...buck54321:dcrdex:txcoininterface2

Or, here are the changes on your branch

https://github.com/buck54321/dcrdex/compare/a5151ea...buck54321:dcrdex:txidinterface